tests/portForwardingServer.spec.ts

```diff
@@ -161,3 +165,26 @@ it('should should not allow to connect when the server does not allow port-forwa
 
   await browserServer.close();
 });
+
+it('should proxy local requests', async ({ getPlaywright, mode, browserName, server, browserOptions }, workerInfo) => {
+  it.skip(mode !== 'service');
```

Comment:
Maybe an own service.spec.ts makes sense.

Reply:
I wonder whether the tests should be agnostic to service/connect, and we could run them in both modes? I'd suggest a helper function right in this file that would either launch the browser in the service mode, or launchServer+connect in default mode. WDYT?